creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -538,6 +582,7 @@ const _issueUpdateReplicaSetOp = async (
         })
 
       if (!canReconfig) {
+        response.result = 'skip_update_replica_set'
         logger.info(
           `[_issueUpdateReplicaSetOp] skipping _updateReplicaSet as reconfig already occurred for userId=${userId} wallet=${wallet}`
         )
```

Comment:
def want to have a specific result for failure from `await _updateReplicaSet()` on 592 - i think you have implicitly done that on line 659 below?